main.go

```diff
@@ -95,6 +96,9 @@ func main() {
 	flag.IntVar(&machineDeploymentConcurrency, "machinedeployment-concurrency", 10,
 		"Number of machine deployments to process simultaneously")
 
+	flag.BoolVar(&kubeadmEnabled, "kubeadm-enabled", true,
```

Comment:
How about `enable-kubeadm-bootstrapper`?

Reply:
Totally fine by me. I was going back and forth on what to name the flag that made the most sense.

Reply:
Fixed in amended commit.